driver/driver.go

```diff
@@ -0,0 +1,633 @@
+package driver
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/jsimonetti/rtnetlink"
+	"github.com/jsimonetti/rtnetlink/internal/unix"
+	"github.com/mdlayher/netlink"
+)
+
+func init() {
```

Comment:
Can you add a comment here that the `driver` package autoloads these drivers on `init` and reference to our conversation earlier in the PR?